pennylane/operation.py

```diff
@@ -900,6 +900,16 @@ def has_decomposition(cls):
             or cls.decomposition != Operator.decomposition
         )
 
+    @property
+    def _pauli_rep(self):
+        """The operator represented as a linear combination of Pauli
+        words using the PauliSentence class.
+
+        Raises:
+            NotImplementedError: if no such representation is provided.
+        """
+        raise NotImplementedError(f"Pauli representation not implemented for {self}.")
```

Comment:
Or would `None` be a better default? Then we wouldn't have to use `try-except` blocks.

Reply:
None as default works much better! Changed in the main integration PR and rebased here !